### PR TITLE
Create Issue Templates

### LINKED ISSUE #9: Fully completed GitHub community profile
So far we need to complete: 

- [x] Description
- [x] README.md (continuously changing) 
- [x] [Code of Conduct](https://github.com/spacebin-for-astronauts/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] CONTRIBUTORS.md ( #8 )
- [x] Issue templates (Bug, Feature Request)
- [x] Pull Request templates

### PR DESCRIPTION
**Pull request by jackiedorland**
Title says it all. Alpha v0.1.

### CODE REVIEW

**Comment by lukewhrit** [comment]
Instead of separate sections for desktop and smartphone, a device section would be sufficient.

**Comment by lukewhrit** [comment]
We also need to make it clear that these issues should only be for the spacebin server, and issues with clients should be made in their own repositories.

> Instead of separate sections for desktop and smartphone, a device section would be sufficient.

I'm thinking now we should just entirely remove these sections and replace with a section for Server OS, etc.

**Comment by jackiedorland** [comment]
> We also need to make it clear that these issues should only be for the spacebin server, and issues with clients should be made in their own repositories.
> 
> > Instead of separate sections for desktop and smartphone, a device section would be sufficient.
> 
> I'm thinking now we should just entirely remove these sections and replace with a section for Server OS, etc.

I agree; people complaining about frontend in here would get old *really* quickly.

**Comment by jackiedorland** [comment]
Hold on, let me write these templates up really quick.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,28 @@
...
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**To Reproduce**    
+Steps to reproduce the behavior (what were you doing at the time?):
+
+-
```
Comment (lukewhrit) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think these should be numbered.

Comment (jackiedorland) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nyalrighty then one second...

Comment (jackiedorland) [reply .github/ISSUE_TEMPLATE/bug_report.md]: New commit should fix. Re-review.

**Review by jackiedorland — COMMENTED** [review]
Go for it!

**Review by lukewhrit — APPROVED** [review]
Looks good to me

**Comment by jackiedorland** [comment]
🎉 :feelsgood:

**Pull request merged**